spec/factories/users.rb

```diff
@@ -2,16 +2,71 @@
   factory :user do
     email { Faker::Internet.email }
     google_id { Faker::Alphanumeric.alphanumeric(number: 10) } # Update with same number of characters as id
```

Comment:
Do we know how many characters the `google_id` is ... if it is a fixed number of characters?

Reply:
21 (according to mine)

Reply:
Awesome -- thank you!  I updated this in the latest commits.